src/interactions/panZoomInteraction.ts

```diff
@@ -8,8 +8,8 @@ export module Interactions {
      */
     private static _PIXELS_PER_LINE = 120;
 
-    private _xScale: QuantitativeScale<any>;
-    private _yScale: QuantitativeScale<any>;
+    private _xScales: QuantitativeScale<any>[];
+    private _yScales: QuantitativeScale<any>[];
```

Comment:
I was highly thinking of using a `Set` but then I run into the situation where I have to turn the `Set` into an array for the getters.  Are we ok with this?